template/src/screens/Home.test.tsx

```diff
@@ -2,11 +2,11 @@ import React from 'react';
 
 import { render, screen } from '@testing-library/react';
 
-import App from './App';
+import HomeScreen from './Home';
```

Comment:
@malparty @tyrro I moved the `App.test.tsx` into `screens/Home.test.tsx` because we just have the tests for HomeScreen for now